suite-native/module-device-settings/src/components/DeviceSettingsCardLayout.tsx

```diff
@@ -16,7 +16,7 @@ type DeviceSettingsCardProps = {
     alertBoxProps?: Omit<AlertBoxProps, 'borderRadius'>;
 };
 
-export const DeviceSettingsCard = ({
```

Comment:
Why the name change?

Reply:
Because the name is misleading, I myself got confused. IMO the new name better reflects the purpose of the class.

Can revert if you don't think it's an improvement.

Reply:
It feels the same or maybe a bit worse to understand, but leave it be :)